javascript-data-model/order-history.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-octal */
+const orderHistory = [
+  {
+    itemOrdered: 'Javascript for impatient programmers',
+    deliveredDate: '2020-08-8',
+    shippedTo: 'JS Masher',
+    orderPlaced: '2020-08-04',
+    additionalInfo: [{ price: 31.55, returnWindow: '2020-09-07' }],
+  },
+  {
+    itemOrdered: 'The Timeless Way of Building',
+    deliveredDate: '2020-07-20',
+    shippedTo: 'JS Masher',
+    orderPlaced: '2020-07-19',
+    additionalInfo: [{ price: 31.55, returnWindow: '2020-08-19' }],
+  },
+  {
+    itemOrdered: 'Gamecube controller adapter',
+    deliveredDate: '2020-07-07',
+    shippedTo: 'JS Masher',
+    orderPlaced: '2020-07-03',
+    additionalInfo: [{ price: 15.98, returnWindow: '2020-08-05' }],
+  },
+  [
```

Comment:
Nice! I like the way you thought this out. 

One thing I would potentially change is have all objects have an `items` property that houses an array of each `item` on the order. I'll go over it in Q&A.
So orders with only one Item would still have an `items` array with only one index in it.